distribution/docs/src/main/resources/_contents/_configuring/configuring-product-cache-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+==== Configuring the Product Cache
+
+All caching properties are part of the <<ddf.catalog.resource.download.ReliableResourceDownloadManager,Resource Download Settings>>.
+
+===== Invalidating the Product Cache
+
+. The product cache directory can be administratively invalidated by turning off the product caching using the cacheEnabled property.
```

Comment:
If this description is for the admin console, this property is labelled `Enable Product Caching` in the UI.

We probably need to decide how these feature configurations should be documented. Should we document each feature/capability and indicate how it can be configured using the admin console, command console, config files, etc., or, for each different configuration approach, discuss each feature/capability. Thoughts?